java/src/main/kotlin/io/spine/validation/java/rule/CompositeRuleGenerators.kt

```diff
@@ -24,7 +24,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.validation.java
+package io.spine.validation.java.rule
```

Comment:
Do I understand correctly this package will eventually die along with all of its contents?

Reply:
Yes.